src/Views/MainView.vala

```diff
@@ -84,6 +85,9 @@ public class SwitchboardPlugUserAccounts.Widgets.MainView : Gtk.Box {
 
         append (paned);
 
+        var settings = new Settings ("io.elementary.settings");
```

Comment:
If you can update the other branches with the schema lookup I'd be happy to merge them :)

Reply:
Yes, I will adjust it in this repository and in the other plugins where I submit 😁️.